changelog/unreleased/kong/bump-dockerfile-ubi9.yml

```diff
@@ -0,0 +1,2 @@
+message: "Bumped rpm dockerfile to UBI 9"
```

Comment:
I would actually suggest to drop this this changelog as we just changed the placeholder value, but user won't expect changes so having this could lead to confusion.

Reply:
Let me update the changelog entry to reflect the fact that I'm changing the default value of the argument. I can practically agurentee that we have folks building using this dockerfile that don't supply any build-args.